selftests/unit/test_utils_partition.py

```diff
@@ -33,7 +33,7 @@ def cannot_sudo(command):
     try:
         process.run(command, sudo=True)
         False
-    except process.CmdError:
+    except (process.CmdError, OSError):
```

Comment:
Well actually I looked at the code and the handling is similar to `subprocess` handling. Also users are probably used to catching the `OSError` so let's keep process as is and only deal with the failure here...